WindowManager.cs

```diff
@@ -259,19 +270,24 @@ public void Initialize(ContentManager content, string contentPath)
             SoundPlayer = new SoundPlayer(Game);
 
             gameWindowManager = new WindowsGameWindowManager(Game);
+#if WINFORMS
             gameWindowManager.GameWindowClosing += GameWindowManager_GameWindowClosing;
+#endif
```

Comment:
Wonder if a good non-WinForms alternative would be subscribing to the `Game.Exiting` event.

Reply:
Yes this seems to work as expected.